Makefile

```
@@ -157,7 +155,7 @@ lint:
 	golint -set_exit_status ./...
 
 mdlint:
-	find docs/ -path docs/vendor -prune -false -o -name '*.md' | xargs $(MDL) -s docs/mdl-style.rb
+	./scripts/test-infra/mdlint.sh
```

I commented (below) that `scripts/test-infra` are supposed to be called by prow only but I think this could be an exception as the environment is supposed to be identical (container) to prow

```suggestion
	${CONTAINER_RUN_CMD} \
	    --rm \
	    --volume "${PWD}:/workdir:ro,z" \
	    --workdir /workdir \
	    ruby:slim \
	    /workdir/scripts/test-infra/mdlint.sh
```